assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+
+[Idea Carla: describe knowledge transfer between subcontractors at HealthCare.gov? Compare against 'tribe' philosophy of Spotify?]
+
+##### Structure
+
+> Structure is about how the organization is arranged or configured. Many organizations take a
+> functional structure, in which different branches of the organization specialize in particular
+> functions, such as manufacturing, sales, and service. Another common form is a divisional
+> structure, in which divisions are arrayed according to geographical regions, products, or some
+> other factor, and each division contains a full complement of functional elements (e.g., sales,
+> service, R&D).
+
+##### Systems
+
+> Systems refers to the rules and regulations, standards, and processes for getting things done and managing the activity of the organization, both formal and informal.
+
+##### Staffing
+
+> Staffing refers to people in the organization and how they are recruited, selected, and trained to
+> perform their jobs. Leadership is included in this category, whether leaders are selected from the
+> outside or developed from within. Staffing addresses the question of how the organization
+> assures it has the right people to do the work.
+
+##### Skills
+
+> Skills are the special, distinctive competencies of the organization, both in the people and the
+> organization itself. What is the organization good at, and what special skills and competencies
+> are found its people?
+
+##### Style
+
+> Style deals mainly with the leadership approach of the top managers in the organization. How do
+> the leaders interface with subordinates and others in the organization, and how do members
+> interact with each other? Style also incorporates the climate or culture of the organization, which
+> can be expressed in terms of job titles, space allocation, workplace architecture, artifacts, and
+> policies.
+
+##### Shared Values
+
+> Shared values are the core beliefs in the organization about what is important and why the
+> organization exists. They are incorporated in formal statements, such as a vision statement, and
+> in informal expressions and practices around the organization. Shared values are the underlying
+> principles or ideas that provide meaning and purpose in an organization.
+
+## 2.4 RUP ##
+### **What can we learn from this about RUP?**
+##### Spotify
+[Please append your findings about Spotify here Carla]
+##### KLM
+Our insights on KLM do not help in dealing with RUP. RUP helps with producing in changing environment. For example, one of the key components in RUP is requirement management, since these are often changed during the project. KLM does not have this problem, customers buy a ticket and get their transportation to their destination. Every part of the job is done according to strict guidelines and rulebooks, software development is more of a creative job where you do not know how things will go at the start of the project.
```

Comment:
"requirement management" => "requirements management" 

This makes it more consistent with everything else.